lib/mparticle_flutter_sdk_web.dart

```diff
@@ -295,6 +295,12 @@ class MparticleFlutterSdkWeb {
           mpCommerce.callMethod('setCurrencyCode', [currency]);
         }
 
+        var eventOptions = {};
+        bool? shouldUploadEvent = commerceEvent['shouldUploadEvent'];
+        if (shouldUploadEvent != null) {
+          eventOptions['shouldUploadEvent'] = shouldUploadEvent;
+        }
```

Comment:
this code block appears to be a dead-end, do we actually send eventOptions?

Reply:
Ah good catch. I think I was confused by `logImpression` and `loPromotion` both not passing custom Attributes or flags. Hopefully its right to add them?